pkg/ccl/streamingccl/streamingest/stream_ingestion_job_test.go

```diff
@@ -69,23 +69,6 @@ func TestTenantStreamingCreationErrors(t *testing.T) {
 		sysSQL.ExpectErr(t, "pq: tenant with name \"foo\" already exists",
 			"CREATE TENANT foo FROM REPLICATION OF source ON $1", srcPgURL.String())
 	})
-	t.Run("destination tenant cannot be online", func(t *testing.T) {
-		sysSQL.Exec(t, "CREATE TENANT bar")
```

Comment:
nit: instead of deleting, could you replace these test cases using the new syntax?

Reply:
the timestamp checks don't exist anymore, since we do our own revert that we know is to the correct timestamp, so there are no longer failed checks to test for.

the service mode check however is still relevant, so I added it to TestTenantStreamingFailback.